great-idea-website/css/index.css

```diff
@@ -63,4 +63,106 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.main-container {
+	width: 85%;
+	margin: auto;
+}
+
+nav {
+	display: inline-block;
+	width: 90%;
+	height: 40px;
+}
+
+nav a {
+	display: inline-block;
```

Comment:
I would take out the display: inline-block on the a tags.  keeping it on the nav should be enough.  a tags can be kept inline.